vspace/src/main/java/edu/asu/diging/vspace/config/PersistenceJPAConfig.java

```diff
@@ -88,6 +88,7 @@ Properties additionalProperties() {
        properties.setProperty("hibernate.hbm2ddl.auto", "update");
        properties.setProperty(
          "hibernate.dialect", "org.hibernate.dialect.MySQL5Dialect");
+       properties.setProperty("hibernate.show_sql", "true");
```

Comment:
let's not enable that always; let's put that in the config configurable through pom